README.md

```diff
@@ -1,8 +1,74 @@
-# Jarivs Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Zaied Zaman . Jarvis Consulting
+
+Self-motivated problem-solver and strong collaborator who always strives for innovative and the most suitable solution. I have started working
+as a Jarvis consultant after the completion of my Master's degree in Computer Science (Research-Based). Apart from my Master's research project,
+I have collaborated with several labs to provide software solutions during my master's. These experiences grew interested and confidence inside
+me to contribute to the data engineering industry. Practicing programming problem-solving and participate in programming contests is one of
+my hobbies alongside reading books on different topics.    
+
+## Skills
+
+**Proficient:** Algorithms & Data structures, Java, Linux/Bash, RDBMS/SQL, Agile/Scrum, Git, Springboot, Maven
+
+**Competent:** C, C++, Image Processing algorithms e.g. BM3D, Image processing tools e.g. Matlab, OpenCV
+
+**Familiar:** Python, JavaScript, Node.js, Express.js, HTML, CSS
+
+## Development Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_demo](https://github.com/jarviscanada/jarvis_data_eng_demo)
+
+- **[Cluster Monitor](./linux_sql)**: Developed a `Bash` and `PostgreSQL` based Cluster monitor agent for distributed systems. The Node machines send continuos hardware usage and hardware specification data which is monitored and persisted by server machine in a PostgreSQL database. The PostgreSQL database instance was hosted by a `docker` container. The container was created based on the PostgreSQL base image from the `docker registry`. `Maven` has
+been used to manage dependencies.
+- **[Core Java Apps](./core_java)**: Developed three `Maven` managed pure `Java`-based apps. 
+    * The Twitter app show, post, delete Tweet data using the Twitter API. Efficient exception handling, logging, and testing frameworks have been used in these apps. `self4j` facade and as the implementation `log4j` was used for the logging purpose whereas `Mockito` framework was used for unit testing and `Junit 4` was used for Integration testing. `MVC` architecture is used where `Dao` pattern is used for data access and retrieval. `Jackson` API is used to parse and create `JSON` objects to `POJO` objects or vice versa.
```

Comment:
stick to present or past. SInce you begin with the past, please continue as "MVC architecture was used.......pattern was used.....Jackson API was used....